stdlib/_locale.pyi

```diff
@@ -0,0 +1,96 @@
+import sys
+from _typeshed import StrPath
+from collections.abc import Iterable, Mapping
+
+LC_CTYPE: int
+LC_COLLATE: int
+LC_TIME: int
+LC_MONETARY: int
+LC_NUMERIC: int
+LC_ALL: int
+CHAR_MAX: int
+
+def setlocale(category: int, locale: str | Iterable[str | None] | None = None) -> str: ...
+def localeconv() -> Mapping[str, int | str | list[int]]: ...
+
+if sys.version_info >= (3, 11):
+    def getencoding() -> str: ...
+
+def strcoll(__os1: str, __os2: str) -> int: ...
+def strxfrm(__string: str) -> str: ...
+
+# native gettext functions
+# https://docs.python.org/3/library/locale.html#access-to-message-catalogs
+# https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Modules/_localemodule.c#L626
+if sys.platform == "linux" or sys.platform == "darwin":
```

Comment:
Are you sure these only exist on linux and darwin, and not on any other unix systems? This is a bit of a smell to me; at a guess, I'd think this would probably be more accurate:

```suggestion
if sys.platform != "win32":
```

Reply:
We need freebsd runner to test this :)
I am fine with both for now.